### PR TITLE
Hotfix for latex docstring

### PR DESCRIPTION
**Pull request by TheCedarPrince**
This should fix the docstring of `latex` from issue #99

### CODE REVIEW

File: src/Javis.jl [hunk 1]
```diff
@@ -541,13 +541,15 @@ latex(text::LaTeXString, action::Symbol) = latex(text, 10, action)
     latex(text::LaTeXString, font_size::Real, action::Symbol)
 
 Add the latex string `text` to the top left corner of the LaTeX path.
-Can be added to `Luxor.jl` graphics such as `Video` or `Drawing`.
+Can be added to `Luxor.jl` graphics such as [`Video`](@ref) or [`Drawing`](@ref).
```
Comment (Wikunia) [inline src/Javis.jl:544]: Does drawing work? I don't think we have a Drawing function.

File: src/Javis.jl [hunk 2]
```diff
@@ -541,13 +541,15 @@ latex(text::LaTeXString, action::Symbol) = latex(text, 10, action)
...
-**It can be installed using the following command
-    (you may have to prefix this command with `sudo` depending on your installation):**
+**NOTES:**
+- **This only works if `tex2svg` is installed.**
+    It can be installed using the following command (you may have to prefix this command with `sudo` depending on your installation):
 
-> `npm install -g mathjax-node-cli`
+        npm install -g mathjax-node-cli
+
+- **The `latex` method must be called from within a [`Action`](@ref).**
```
Comment (Wikunia) [inline src/Javis.jl:552]: *an Action?

File: src/Javis.jl [hunk 3]
```diff
@@ -566,11 +568,21 @@ Available actions:
...
-finish()
+function ground(args...)
+    background("white")
+    sethue("black")
+end
+
+function draw_latex(video, action, frame)
+    latex(
+	L"\\sqrt{5}",
+        50,
```
Comment (Wikunia) [inline src/Javis.jl:579]: Maybe add a comment that this is for font size? With an inline comment

**Review by Wikunia — CHANGES_REQUESTED** [review]
Looks good in general just some minor things with maybe a Documenter.jl error

File: src/Javis.jl (hunk 2 above)
Comment (TheCedarPrince) [reply src/Javis.jl:552]: Grammar :sweat_smile:

**Review by Wikunia — APPROVED** [review]
Lgtm

**Pull request merged**